### DISCUSSION TITLE
[Tests] Setup Istanbul Test Coverage tool

### DISCUSSION THREAD

**Discussion by kalimar** [Terrastories/terrastories#728] [Temporarily Archived Tickets]
https://istanbul.js.org/

**Comment by GQuirino** [comment]
Can i take a look into this issue?

**Comment by kalimar** [comment]
@GQuirino we made some requirement changes but feel free to continue working on this. https://github.com/Terrastories/terrastories/milestone/7 - You can configure Istanbul within Jest. I tried a few different things but no luck - test coverage reports came up empty. I'm pretty sure it's one or two of the configuration options in `package.json`
